src/qt/forms/sendcoinsdialog.ui

```diff
@@ -848,7 +848,7 @@
                  <item>
                   <widget class="QLabel" name="labelCustomPerKilobyte">
                    <property name="toolTip">
-                    <string>For the transaction fee, you can specify the exact number of "satoshis per kilobyte (1,000 bytes)."  For transactions larger than 1 kilobyte, the fee will be calculated on a per-kilobyte basis.  For transactions smaller than 1 kilobyte, the fee will be apportioned according to the size of the transaction (e.g., for a transaction that is 500 bytes {half of 1 kilobyte}, a fee of 0.00000010 would therefore be reduced to 0.00000005).</string>
+                    <string>Specify a custom fee per 1,000 bytes of virtual transaction size.</string>
```

Comment:
This is just my opinion, but we shouldn't need to specify that the size is virtual. Either the user knows what virtual means, so they know that all sizes are virtual. Or they don't, in which case it just adds confusion.

Reply:
I know that there are two sizes: virtual and actual. I didn't know (until reading the source) which of the two this custom setting was specifying. It would have been useful for me if the tooltip had cleared up the ambiguity.

When I double-click on a transaction in the transaction list it shows the actual size and not the virtual size. That's the only other place I've seen any mention of transaction size in the GUI.

Reply:
> It appears that @dooglus' suggestion disappeared since I added the new commit.

I didn't remove anything. You may need to click "view outdated" since my comment was in response to an old commit.

Reply:
I think that should probably be changed to virtual size.